src/modules/imageresizer/ui/Views/InputPage.xaml

```diff
@@ -6,62 +6,53 @@
     xmlns:p="clr-namespace:ImageResizer.Properties"
     xmlns:ui="http://schemas.lepo.co/wpfui/2022/xaml">
 
-    <Grid>
+    <Grid Name="RootGrid">
```

Comment:
Do we use this name anywhere in code? If not, let's leave it out as it's confusing. If so, use `x:Name` to be consistent